services/ui-src/src/components/forms/Form.tsx

```diff
@@ -56,6 +56,11 @@ export const Form = ({
 
   const reportWithSubmittedStatus = reportStatus === ReportStatus.SUBMITTED;
 
+  // For the SAR RE&T sections, display an alert if the MFP WP does not contain any target populations
+  const displayRetError =
+    report?.populations?.length === 0 &&
+    location.pathname.startsWith("/sar/recruitment-enrollment-transitions");
```

Comment:
I'm getting `false` for this. Specifically the length of populations comes back as `4`. Seems like it's still counting them even if they are selected as "no". What are others experiencing? 


I used the postman collection to create the WP and SAR.

Reply:
I'm taking a look at that now! 👀

Reply:
@gmrabian can you try this again? I'm not able to replicate your issue, maybe the Postman collection was adding target populations?

Reply:
Bug went away 🤷